### PR TITLE
config-local-debugging-and-refactor

### PR DESCRIPTION
**Pull request by urirot**
- replace print with logger
- add config file to run local and docker-compose
- update readme

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Set environment variable: `export IS_LOCAL=True`
+- Set environment variable: `export FLASK_PORT=3333` we need it to be a different port then the one in the docker-compose
+- Run app.py
+#### Run docker - before pushing your code
+- Install docker - `https://docs.docker.com/install`  
+
+_Compose instructions_  
+- Install Docker Compose - `https://docs.docker.com/compose/install/`      
+- Most package managers have it as `docker-compose` and it's largely just a shell script.    
+- `docker compose -d up` to bring up the database and the server.
```
Comment (sposerina) [inline src/README.md:16]: should be docker-compose

File: src/config.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import os
+
+# Determine if app is ran from docker or local by env var IS_LOCAL
+BASE_PATH = '/app/static/'
+DB = 'postgresql://postgres:thispasswordisverysecure@paws-compose-db/postgres'
+
+IS_LOCAL = os.getenv('IS_LOCAL', False)
+
+if IS_LOCAL:
```
[the remark this answers is not in the document]
Comment (urirot) [reply src/config.py:9]: sure, adding the init file wouldn't harm. are you sure it's a windows issue and not python version? did you run it with python3?

I like the idea of adding a configuration for the DB ip address

File: src/README.md (hunk 1 above)
Comment (urirot) [reply src/README.md]: updated

File: src/config.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (urirot) [reply src/config.py:9]: updated. can you check if you need the innit.py with python3.6+? 
if you still need it, can you please push it to this branch before we merge?

[the remark this answers is not in the document]
Comment (sposerina) [reply src/config.py:9]: sure, I'll test

**Pull request merged**